src/dev-spaces-preview/azext_dev_spaces_preview/custom.py

```diff
@@ -38,11 +38,11 @@ def ads_use_dev_spaces(cluster_name, resource_group_name, space_name='default',
 
     from subprocess import PIPE
     retCode = subprocess.call(
-        [azds_cli, 'resource', 'select', '-n', cluster_name, '-g', resource_group_name],
+        [azds_cli, 'controller', 'select', '--name', cluster_name, '--resource-group', resource_group_name],
         stderr=PIPE)
     if retCode != 0:
         retCode = subprocess.call(
-            [azds_cli, 'resource', 'create', '--aks-name', cluster_name, '--aks-resource-group',
+            [azds_cli, 'controller', 'create', '--target-name', cluster_name, '--target-resource-group',
              resource_group_name, '--name', cluster_name, '--resource-group', resource_group_name],
```

Comment:
This passes resource_group_name for --target-resource-group. Do we want to support creating the Controller in a different resource group than the cluster? If so, we'll need to introduce a target_resource_group_name param to pass in.

Reply:
For now, this is the spec for az cli extension we have for the use and remove command. We can update this once we have the use and remove command in azds cli. @stepro Would you like to comment on this one?